server/core/src/main/java/dev/slimevr/tracking/trackers/Tracker.kt

```diff
@@ -327,6 +330,8 @@ class Tracker @JvmOverloads constructor(
 			filteringHandler.getTrackedRotation()
 		}
 
+		rot = EulerAngles(EulerOrder.YZX, 0.0f, yawCorrectionInRad, 0.0f).toQuaternion() * rot
```

Comment:
`EulerAngles(EulerOrder.YZX, 0.0f, yawCorrectionInRad, 0.0f).toQuaternion()` should be cached on changes to `yawCorrectionInRad`.

Reply:
Also, this needs to be applied *after* reset and mounting rotations

Reply:
Addressed by updating a private variable `spineYawCorrectionRot` when `spineYawCorrectionInRad` is set.